src/TicketingSystem.WebApi/Controllers/VenuesController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using TicketingSystem.BusinessLogic.Services;
+
+namespace TicketingSystem.WebApi.Controllers
+{
+    [ApiController]
```

Comment:
We could put all the controllers into one WebApi project in scope of this course. This would safe you lots of time with development and testing for just creating copies of common files like Program.cs, settings ... And it would be easier to demo - just run one API with all the controllers in swagger in place. You can choose what ever approach, but remove duplicates then.

Reply:
Merged API projects into a single WebApi project